lib/cryptography.js

```diff
@@ -33,13 +39,18 @@ module.exports.encrypt = (options) => {
     const secret = options.secret || 'mySecret'
     const inputFile = options.inputFile || '.env'
     const outputFilePath = options.outputFile || `${inputFile}.enc`
-    const encryptionAlgo = options.encryptionAlgo || 'aes192'
+    const encryptionAlgo = options.encryptionAlgo || 'aes256'
+    const ivLength = options.ivLength || 16
+    // presumably createCipheriv() should work for all the algo in ./openssl_list-cipher-algorithms.csv with the right key/iv length
```

Comment:
I assume this is with reference to the gist: https://gist.github.com/reggi/4459803. 

Please mention the source link over here and remove the file `./openssl_list-cipher-algorithms.csv` from `lib` folder, as the file is not used anywhere within our code and it's only there for reference purpose.

Reply:
this is the output of the shell command 
`$ openssl list-cipher-algorithms`
depending on the installed version of openSSL, more or less algorithms will (probably) be available
note with openSSL 1.1.1 i have to use 
`$ openssl enc -ciphers` to get a list
https://github.com/kunalpanchal/secure-env/issues/2 might be related to this, but i can only speculate